src/status_im/ui/screens/privacy_and_security_settings/views.cljs

```diff
@@ -89,3 +103,19 @@
           :on-press            #(re-frame/dispatch [:navigate-to :delete-profile])
           :accessibility-label :dapps-permissions-button
           :chevron             true}])]]))
+
+(defn radio-item [id value]
+  [quo/list-item
+   {:active    (= value id)
+    :accessory :radio
+    :title     (get titles id)
+    :on-press  #(re-frame/dispatch [:multiaccounts/profile-picture-show-to-switched id])}])
+
+(views/defview profile-pic-show-to []
+  (views/letsubs [{:keys [profile-pictures-show-to]} [:multiaccount/profile-pictures-show-to]]
```

Comment:
Here  you are destructuring an object, but `[:multiaccounts/profile-pictures-show-to]` returns an integer,
so it should simply be:
```
profile-pictures-show-to [:multiaccount/profile-pictures-show-to]
```